src/lib/plugins/moderation.js

```diff
@@ -1,12 +1,11 @@
-const moment = require('moment')
-const needle = require('needle')
+const https = require('https')
 const UserError = require('flying-squid').UserError
 
 module.exports.server = function (serv, settings) {
   serv.ban = async (uuid, reason) => {
     if (!serv.bannedPlayers[uuid]) {
       serv.bannedPlayers[uuid] = {
-        time: +moment(),
+        time: +(Date.now() / 1000 + '').replace(/\./, ''),
```

Comment:
this looks pretty bad

Reply:
I don't think he realize this code is equivalent to just `Date.now()` ?